### ISSUE TITLE
Not working on Realme C25Y

### ISSUE THREAD

**Issue by mesflit** [KieronQuinn/ClassicPowerMenu#135]
Android 11
root enabled
xposed enabled
but not working

**Comment by KieronQuinn** [comment]
Disable the Xposed module and use the accessibility option. If that doesn't work, nothing I can do.
